pallets/elections/src/migration.rs

```diff
@@ -1,162 +0,0 @@
-use codec::{Decode, Encode};
-use frame_election_provider_support::Weight;
-use frame_support::{
-    log,
-    pallet_prelude::{StorageVersion, TypeInfo},
-    traits::OnRuntimeUpgrade,
-};
-use primitives::CommitteeSeats;
-use sp_core::Get;
-#[cfg(feature = "try-runtime")]
-use {frame_support::ensure, pallets_support::ensure_storage_version, sp_std::vec::Vec};
-
-use crate::{CommitteeSize, Config, NextEraCommitteeSize, Pallet, LOG_TARGET};
```

Comment:
I don't think we should remove migrations from our code. We should just detach them from the runtime...

Reply:
That also I thought, yet I don't see now in `aleph-node` repo any other migrations, hance I thought to follow that rule and remove them. Let me check what substrate does with that

Reply:
Yes, the reason is we had this pallet hook based migrations before, which were removed to clean things up. But it seems the convention is to keep migrations, and just attach them when needed.

Reply:
I have removed them quite some time ago. We have them in the git history if they will be ever needed (?). I have removed them because maintaining the old migrations is kinda meaningless and makes it harder to develop pallets.

Reply:
Actually, in Substrate they indeed keep migrations. I'll will bring them back then